packages/plugin-ext/package.json

```diff
@@ -29,6 +29,7 @@
     "connect": "^3.7.0",
     "decompress": "^4.2.0",
     "escape-html": "^1.0.3",
+    "fs-extra": "^4.0.2",
```

Comment:
There is no need to add `fs-extra` or `@types-fs-extra` since they are pulled from `@theia/core` (the root extension for all extensions). There is an additional `tsconfig` rule to make dependencies exceptions for the `no-implicit-dependency` rule:

https://github.com/eclipse-theia/theia/blob/9fb6351621583f4a3e51954debbea9ed1e080168/packages/core/package.json#L14

https://github.com/eclipse-theia/theia/blob/9fb6351621583f4a3e51954debbea9ed1e080168/packages/core/package.json#L29

https://github.com/eclipse-theia/theia/blob/9fb6351621583f4a3e51954debbea9ed1e080168/configs/errors.tslint.json#L59

The corresponding pull-request: https://github.com/eclipse-theia/theia/commit/f9ff23786ed48f611aa335527c78db70427edc54

Reply:
done